app/services/audits/audit_service.rb

```diff
@@ -6,8 +6,12 @@ def initialize(school)
       @school = school
     end
 
+    def last_audit
+      @school.audits.published.order(created_at: :desc).first
+    end
+
     def recent_audit
-      @school.audits.where("created_at >= ?", 90.days.ago).order(created_at: :desc).first
+      @school.audits.published.where("created_at >= ?", 90.days.ago).order(created_at: :desc).first
```

Comment:
Now I've seen the other syntax, should we be using: `.where(created_at: 90.days.ago...)`? Although easy to miss the double dots I guess.

Reply:
Yes - I'll change it now. Might just be 2 dots though for >= as 3 dots would be >